tests/meta/orders/test_external_data.py

```diff
@@ -30,9 +32,9 @@ def test_create_source_path(external_data_api: ExternalDataAPI):
     """Test generating the source path"""
 
     # WHEN the source path is created
-    source_path = external_data_api.create_source_path(
-        ticket_id=999999, raw_path="/path/%s", cust_sample_id="ABC123", cust_id="cust000"
-    )
+    external_data_api.ticket_id = 999999
+    external_data_api.customer = "cust000"
```

Comment:
Is there no test fixture for customer? Otherwise we should create it and reuse it